lightkube/generic_resource.py

```diff
@@ -1,7 +1,9 @@
 from typing import Type, Any, Optional, overload
 
 from .core import resource as res
+from .core.client import Client
 from .core.internal_models import meta_v1, autoscaling_v1
+from .resources.apiextensions_v1 import CustomResourceDefinition
 
 __all__ = ['create_global_resource', 'create_namespaced_resource']
```

Comment:
I guess we need to expose this new public functions here.

Reply:
Done